python/templates/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 set(PODIO_TEMPLATES
   ${CMAKE_CURRENT_LIST_DIR}/Collection.cc.jinja2
   ${CMAKE_CURRENT_LIST_DIR}/Collection.h.jinja2
+  ${CMAKE_CURRENT_LIST_DIR}/AllCollections.h.jinja2
```

Comment:
Only nitpicking comment would be to change the name of the jinja2 template as well

Reply:
done.